File: packages/expo-dev-menu/vendored/react-native-safe-area-context/android/devmenu/com/th3rdwave/safeareacontext/EdgeInsets.java
package devmenu.com.th3rdwave.safeareacontext;

/* package */ class EdgeInsets {
  float top;
  float right;
  float bottom;
  float left;

  EdgeInsets(float top, float right, float bottom, float left) {
    this.top = top;
    this.right = right;
    this.bottom = bottom;
    this.left = left;
  }

  boolean equalsToEdgeInsets(EdgeInsets other) {
    if (this == other) {
      return true;
    }
    return this.top == other.top && this.right == other.right && this.bottom == other.bottom && this.left == other.left;
  }
}

File: packages/expo-dev-menu/vendored/react-native-safe-area-context/android/devmenu/com/th3rdwave/safeareacontext/EdgeInsets.kt
package devmenu.com.th3rdwave.safeareacontext

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

data class EdgeInsets(
  @Field var top: Float,
  @Field var right: Float,
  @Field var bottom: Float,
  @Field var left: Float
) : Record

